LayoutTests/accessibility/mac/scrolling-in-pdf-crash.html

```diff
@@ -16,8 +17,9 @@
 
     var pageOne, pageTwo;
     var pdfAxObject, pdfLayerController;
-    requestAnimationFrame(async function() {
+    internals?.registerPDFTest(async () => {
```

Comment:
I _think_ you need `window?.internals` to make this robust in the case where `window.internals` isn't defined. Although, simply using `internals.registerPDFTest` is fine too since that should always exist when running these layout tests.

Reply:
Yes, that's fair. I will just fall back to `internals.registerPDFTest`